.github/workflows/eslint.yml

```
@@ -0,0 +1,32 @@
+name: ESLint for Javascript
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP with PECL extension
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+
+      - name: Install dependencies
+        run: composer install --no-progress -q
+
+      - name: Install node modules
+        run: yarn install --silent
+
+      - name: Run ESLint
+        run: yarn run eslint
```

We can add `--quiet` so warnings don't show up when running the action/check.
```suggestion
        run: yarn run eslint --quiet
```